src/AppBundle/Service/InterviewManager.php

```diff
@@ -217,6 +228,61 @@ public function sendInterviewScheduleToInterviewer(User $interviewer)
         $this->mailer->send($message);
     }
 
+    public function sendAcceptInterviewReminders()
+    {
+        $interviews = $this->em->getRepository('AppBundle:Interview')->findAcceptInterviewNotificationRecipients(new \DateTime());
+        /** @var Interview $interview */
+        foreach ($interviews as $interview) {
+            $oneDay = new \DateInterval('P1D');
+            $now = new \DateTime();
+            $moreThan24HoursSinceScheduled = $now->sub($oneDay) > $interview->getLastScheduleChanged();
+            if ($interview->getNumAcceptInterviewRemindersSent() < self::MAX_NUM_ACCEPT_INTERVIEW_REMINDERS_SENT && $moreThan24HoursSinceScheduled) {
+                $this->sendAcceptInterviewReminderToInterviewee($interview);
+            }
+        }
+    }
+
+    private function sendAcceptInterviewReminderToInterviewee(Interview $interview)
+    {
+        $message = \Swift_Message::newInstance()
+            ->setSubject('Påminnelse om intervjuinvitasjon med Vektorprogrammet')
+            ->setTo($interview->getUser()->getEmail())
+            ->setBody(
+                $this->twig->render(
+                    'interview/accept_interview_reminder_email.html.twig',
+                    array(
+                        'interview' => $interview,
+                    )
+                ),
+                'text/html'
+            );
+
+        $this->mailer->send($message);
+
+        $interview->incrementNumAcceptInterviewRemindersSent();
+        $this->em->persist($interview);
+        $this->em->flush();
+
+        $maxNum = self::MAX_NUM_ACCEPT_INTERVIEW_REMINDERS_SENT;
+        $this->logger->info("
+            Accept interview reminder sent to {$interview->getUser()}.
+            ({$interview->getNumAcceptInterviewRemindersSent()}/$maxNum reminders sent)");
+
+
+        $oneDay = new \DateInterval('P1D');
+        $now = new \DateTime();
+        $lessThan24HoursUntilInterview = $now->add($oneDay) > $interview->getScheduled();
+        if ($lessThan24HoursUntilInterview) {
+            $smsMessage = "Hei! Vi har satt opp intervjutid for deg og trenger å vite om den passer. Gå til https://vektorprogrammet.no{$this->router->generate('interview_response', ['responseCode' => $interview->getResponseCode()])} for å se intervjutiden og svare. Mvh Vektorprogrammet.";
```

Comment:
For å gjøre meldingen enda enklere å lese kan du vurdere om du vil legge inn linjeskift, f.eks:

```php
$smsMessage = 
"Hei $name!\n".
"Vi har satt opp intervjutid for deg og trenger å vite om den passer.\n\n".
"Gå til https://vektorprogrammet.no{$this->router->generate('interview_response', ['responseCode' => $interview->getResponseCode()])} for å se intervjutiden og svare.\n\n".
"Mvh Vektorprogrammet.";
```